cryptus-app/components/market_overview/sort_button/sort_button.module.scss

```diff
@@ -0,0 +1,23 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
+
+.container {
+  flex: 0 1 auto;
+  gap: 4px;
+  width: min-content;
+  height: 32px;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  border-radius: 16px;
+  background: #ebe7e3;
+}
+.text {
+  padding-right: 8px;
+  color: #767472;
+  padding-left: 4px;
```

Comment:
use global for color, and use combined padding syntax `padding: 0px 8px 0px 4px`